### PR TITLE
refactor: color variables, added assets

### PR DESCRIPTION
**Pull request by SinchanaMS**
Refactored the CSS variables with block letters.
In Component's CSS, the variables have been updated.
Added Assets.

### CODE REVIEW

File: stylesheets/documentation.css [hunk 1]
```diff
@@ -262,53 +266,53 @@
...
+    background: var(--GRID-3-SECTION);
 }
 
 .grid-wrapper {
     width: 80%;
     margin-bottom: 2rem;
-    background: var(--grid-wrapper);
-    outline: 2px solid var(--secondary-color);
+    background: var(--GRID-WRAPPER);
+    outline: 2px solid var(--SECONDARY-COLOR);
```
Comment (ShrutiLakhani) [inline stylesheets/documentation.css:315]: LGTM!

**Review by ShrutiLakhani — APPROVED** [review]
Code looks clean! Good to merge!

**Review by j24m — APPROVED** [review]
Amazing, 5stars for clean code and proper class naming conventions, Just one thing please make small PRs and include less number of files per PR, otherwise everything is great.

**Review by aiman-ansari — APPROVED** [review]
As always your code is very neat  naming convention is also good.
Spacing is also good . I couldn't find any issue in it :)

**Comment by SinchanaMS** [comment]
Thank you very much for the review! 
Merging to dev.

**Pull request merged**